cms/check_rules_count_volume_per_rse_activity_state

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+# Copyright 2012-2024 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Christos Emmanouil, <christos.emmanouil@cern.ch>, 2024
+
+"""
+Probe to check number and volume of rules per rse, activity and state.
+Handling rules in the states OK, REPLICATING, STUCK.
+"""
+
+import sys
+import traceback
+
+from sqlalchemy import func, case
+from sqlalchemy.orm import aliased
+
+from rucio.db.sqla import models
+from rucio.db.sqla.session import get_session
+from rucio.common.types import InternalScope
+from rucio.db.sqla.constants import RuleState, DIDType
+
+from utils import common
+PrometheusPusher = common.PrometheusPusher
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if __name__ == "__main__":
+
+    try:
+
+        session = get_session()
+
+        '''
+        # Handling WAITING_APPROVAL, INJECT, SUSPENDED with unique RSE destination
+        # - The information is retried using the following tables in the specified order: rules -> contents -> dids.
+        # - For these states volume information can be retrieved from contents and dids table.
+        #   Although, only rules with container dids are counted. If a rule is created using a did of a different type it will be ignored.
+        # - For rules in this state, the information about the destination rse is (only maybe?) available at the rules table (ReplicationRule) under rse_expression.
```

Comment:
Please break up this and all the other long lines.

Reply:
Thanks to @ericvaandering for the feedback.
I’ve reformatted the comments by breaking up the long lines. Please let me know if anything else is needed.